mocktracer/mocklogrecord.go

```diff
@@ -0,0 +1,105 @@
+package mocktracer
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/opentracing/opentracing-go/log"
+)
+
+// MockLogRecord represents data logged to a Span via Span.LogFields or
+// Span.LogKV.
+type MockLogRecord struct {
```

Comment:
The `opentracing.LogRecord` is not very useful for tests, so I went with this... Note that `MockKeyValue` coerces all values to strings but does remember the `reflect.Kind`.